photon-api/src/main/scala/com/linkedin/photon/ml/algorithm/FactoredRandomEffectCoordinate.scala

```diff
@@ -260,22 +259,18 @@ object FactoredRandomEffectCoordinate {
    *
    * @param localDataSetRDD The training dataset
    * @param modelsRDD The individual random effect models
-   * @param sparsityToleranceThreshold If the product between a certain feature and coefficient is smaller than
-   *                                   sparsityToleranceThreshold, then it will be stored as 0 for sparsity
-   *                                   consideration
    * @return Kronecker product result
    */
   private def kroneckerProductFeaturesAndCoefficients(
       localDataSetRDD: RDD[(String, LocalDataSet)],
-      modelsRDD: RDD[(String, GeneralizedLinearModel)],
-      sparsityToleranceThreshold: Double = 0.0): RDD[(Long, LabeledPoint)] = {
+      modelsRDD: RDD[(String, GeneralizedLinearModel)]): RDD[(Long, LabeledPoint)] = {
 
     localDataSetRDD.join(modelsRDD).flatMap { case (_, (localDataSet, model)) =>
       localDataSet.dataPoints.map { case (uniqueId, labeledPoint) =>
         val generatedFeatures = VectorUtils.kroneckerProduct(
           labeledPoint.features,
           model.coefficients.means,
-          sparsityToleranceThreshold)
+          MathConst.SPARSITY_THRESHOLD)
```

Comment:
They are logically distinct things, but it could be that `EPSILON` would be a good rule of thumb for a default `SPARSITY_THRESHOLD`. I think we should try it and see what the effect on model size is in the real world.